### DISCUSSION TITLE
Your host PC GPU doesn't support HEVC

### DISCUSSION THREAD

**Discussion by Barkfin** [LizardByte/community#282] [Sunshine]
(Incidentally: both servers work 100% with Parsec, problem is that Parsec doesn't work on Win7 anymore...)

I have two identical installations of Proxmox VE 7.4 on two separate servers (they are cheap these days...)
Each server has its own Tesla P4 video card, and each has a Windows 10 virtual machine.

This implementation works:
On server 1 I installed nVidia GRID host driver V15.3, this is "NVIDIA-Linux-x86_64-525.125.03-vgpu-kvm.run".
For windows 10 on server 1 I installed nVidia GRID guest driver V15.2, "528.89_grid_win10_win11_server2019_server2022_dch_64bit_international.exe".
Now, in Moonlight 4.3.1, Settings/Advanced Settings I have set up Video decoder: Automatic, and Video codec: HEVC (H.265).
And in Settings/Basic Settings I have Video bitrate: 34.5 Mbps (over gigabit ethernet).
What do you know, looks and works fantastic! Crystal clear image, terrific latency, just stellar.
Problem.....   <-------------- the last nVidia GRID guest driver V10.4, that still works on Windows 7, does not work properly with that GRID host driver V15.3.

This implementation doesn't work:
On server 2 I installed nVidia GRID host driver V11.13, this is "NVIDIA-Linux-x86_64-450.248.03-vgpu-kvm.run".
For windows 10 on server 2 I installed nVidia GRID guest driver V11.13, "454.23_grid_win10_server2016_server2019_64bit_international.exe".
Sadly, when I use Moonlight 4.3.1 to connect to Win 10 on server 2, I get error "Your host GPU doesn't support HEVC. A GeForce GTX 900-series (Maxwell) or later GPU is required for HEVC streaming."

Note: both Win10 virtual machines on both servers work just fine with Rustdesk, it's just that Rustdesk doesn't seem to have the H.265 hardware encoding implemented as well. So whereas the image looks really good and latency is "fair," it's nowhere near as good as Sunshine 0.20/Moonlight 4.3.1.

**Comment by Kane-Kuroneko** [comment]
I’m experiencing the same issue on my notebook with the Nvidia 2070s GPU; my Moonlight client is displaying the error message,"Your PC does not support HEVC encoding."

[a comment by an automated account is not shown]
